### PR TITLE
two minor systemctl memleak fixes

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -184,6 +185,85 @@ CHANGES WITH 232 in spe
...
+          entirely.
+
+        * In .netdev files two new settings RemoteChecksumTx= and
+          RemoteChecksumRx= are now understood that permit configuring the
+          remote checksumming logic for VXLAN networks.
+
+        * The service manager learnt a new "invocation ID" concept for invoked
+          services. Each runtime cycle of a service will get a new invocation
+          ID (a 128bit random UUID) assigned that identifies the specific
+          runtime of the service uniquely and globally. A new invocation ID is
```
Comment (martinpitt) [inline NEWS]: "runtime" sounds a bit weird to me -- s/specific runtime/current run/ perhaps?

File: NEWS [hunk 2]
```diff
@@ -184,6 +185,85 @@ CHANGES WITH 232 in spe
...
+          generated each time a service starts up. The journal will store the
```
Comment (martinpitt) [inline NEWS]: "being" generated

File: NEWS [hunk 3]
```diff
@@ -184,6 +185,85 @@ CHANGES WITH 232 in spe
...
+          invocation ID of a service along with any logged messages, thus
+          making the invocation ID useful for matching the online runtime of a
+          service with the offline log data it generated in a safe way without
+          relying on synchronized timestamps. In mayn ways this new service
```
Comment (martinpitt) [inline NEWS]: "mayn" typo

File: NEWS [hunk 4]
```diff
@@ -184,6 +185,85 @@ CHANGES WITH 232 in spe
...
+        * The PAM configuration fragment file for "user@.service" shipped with
+          systemd (i.e. the --user instance of systemd) has been stripped to
+          the minimum necessary to make the system boot. Previously, it
+          contained Fedora-specific stanzas that did not apply to other
+          distributions. It is expected that downstream distributions add
+          additional configuration lines, matching their needs to this file,
+          using it only as rough template of what systemd itself needs. Note
+          that this reduced fragment does not even include an invocation of
+          pam_limits which most distributions probably want to add, even though
+          systemd itself does not need it.
```
Comment (martinpitt) [inline NEWS]: You can also entirely disable the installation of this upstream configuration file with the "--with-pamconfdir=no" configure option if you install your own.

File: NEWS (hunk 2 above)
Comment (keszybz) [reply NEWS]: Actually is sounds better without "being".

**Comment by poettering** [comment]
force pushed a new, rebased version, addressing the NEWS issues. please have a look.

**Pull request merged**